climakitae/selectors.py

```diff
@@ -162,13 +213,52 @@ def view(self):
         proj = ccrs.Orthographic(-118, 40)
         crs_proj4 = proj.proj4_init  # used below
         ax = fig0.add_subplot(111, projection=proj)
-        ax.set_extent([-160, -84, 8, 68], crs=ccrs.PlateCarree())
+        ax.set_extent([-150, -88, 8, 66], crs=ccrs.PlateCarree())
+        ax.set_facecolor("grey")
+
+        def _plot_wrf_domains(ax):
+            """
+            Plots the boundaries of the WRF domains on an existing set of axes for a map.
+            Used with the area selection preview panel in 'select'. We could load some data
+            to do this procedurally, but hard-coding these numbers makes it faster.
+            """
+            _colors = ["k", "dodgerblue", "darkorange"]
+            for i, domain in enumerate(["45 km", "9 km", "3 km"]):
+                # Plot domain
+                ax.add_geometries(
+                    [self._wrf_bb[domain]],
+                    crs=ccrs.PlateCarree(),
+                    edgecolor=_colors[i],
+                    facecolor="white",
+                )
+
+        _plot_wrf_domains(ax)
         ax.coastlines()
+        ax.add_feature(cfeature.BORDERS)
         ax.add_feature(cfeature.STATES, linewidth=0.5)
-
+        ax.annotate(
+            "45-km grid",
+            xy=(-154, 33.8),
+            rotation=28,
+            xycoords=ccrs.PlateCarree()._as_mpl_transform(ax),
```

Comment:
Noted about the multiple instantiation... but this needs to be done on the xy-coords, since those are the coordinate dimensions the data is stored on.